task-ballerina/src/task/commons.bal

```diff
@@ -65,6 +65,18 @@ public type TimerConfiguration record {|
     TimerMisfirePolicy misfirePolicy = "smartPolicy";
 |};
 
+# Configurations related to a ThreadPool.
+#
+# + threadCount - Specifies the number of threads that are available for the concurrent execution of jobs.
+#                 It can be set to a positive integer between 1 and 10. Values greater than 10 are allowed but
+#                 might be impractical.
```

Comment:
Since we panic if the user provides thread count outside the range. shall we specify those values are not allowed?